### ISSUE TITLE
ButtonsModel

### ISSUE THREAD

**Issue by bennlich** [backspaces/agentscript#49]
Currently, only the largest cluster of buttons is indicated by color. It would make the formation of groups easier to see if every cluster was assigned a unique color. Then you would be able to see the distribution of groups more clearly.

**Comment by stephenguerin** [comment]
Yes, that would be visually interesting. This model is based on Stu Kauffman's buttons and thread and discussing phase transition in the size. When we have plotting that will help tell the story on this model. BTW, here's the Netlogo equivalent
https://www.netlogoweb.org/launch#https://www.netlogoweb.org/assets/modelslib/Sample%20Models/Networks/Giant%20Component.nlogo

**Comment by stephenguerin** [comment]
Somewhat related is how to find clusters.

https://www.netlogoweb.org/launch#https://www.netlogoweb.org/assets/modelslib/Code%20Examples/Patch%20Clusters%20Example.nlogo

Note how this model is in the "Code Examples" directory of the models library which is one of my favorite directories as it focuses more on computational motifs that leverage ABM semantics and less about the models representing something.

I think we should have a similar type of motif area.

**Comment by backspaces** [comment]
Sorta like gists for AS!

I'm assuming docs are the highest priority for now tho, right?

**Comment by stephenguerin** [comment]
on Priority, I put GISTS over in the Adjacent Possible Column:
https://github.com/backspaces/agentscript/projects/1#card-61641771
